modules/optimization/include/postprocessors/DiffusionVariableIntegral.h

```diff
@@ -0,0 +1,30 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#pragma once
+
+#include "ElementIntegralPostprocessor.h"
+
+class DiffusionVariableIntegral : public ElementIntegralPostprocessor
```

Comment:
docstring

what does this do really
DiffusionVariableIntegral isnt descriptive

Reply:
It solves eqn 33 on this page:  https://mooseframework.inl.gov/isopod/getting_started/InvOptTheory.html

I'll rename it.